### ISSUE TITLE
[Question] What is the recommended setting for web.config file

### ISSUE THREAD

**Issue by TheBlueSky** [aspnet/AspNetCore#12606]
We're using ASP.NET Core 2.1 and I'm wondering which of the following is the recommended configuration for the web.config file:

Is it this

`<aspNetCore processPath="dotnet" arguments=".\MyApp.dll" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout.log" />`

or this

`<aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout.log" />`

In other words, is it using `dotnet` and the application DLL name specifically, or using the variables `LAUNCHER_PATH` and `LAUNCHER_ARGS`?

**Comment by jkotalik** [comment]
We usually prefer the second one when adding a web.config to your codebase as Visual Studio is able to control that. For example. the values of processPath and arguments may need to be different based on whether you are trying to run the app with IISExpress or with IIS. If you hardcode dotnet and 'MyApp.dll', VS can't control it anymore.  Do note that LAUNCHER_PATH and LAUNCHER_ARGS should not be in your published output. VS will replace those value on publish.

**Comment by TheBlueSky** [comment]
Thanks, @jkotalik.
